client/src/components/auth/login.js

```diff
@@ -62,14 +63,15 @@ class Login extends Component {
   }
 
   _confirm = async data => {
-    const { token } = this.state.login ? data.tokenAuth.token : data.signup;
+    const { token } = data.tokenAuth;
```

Comment:
Why was the ternary removed?

Reply:
Because the ternary was for if this component was handling both login and signup. It is not.